src/applications/mhv-medical-records/api/MrApi.js

```diff
@@ -67,6 +69,25 @@ export const getLabsAndTests = async () => {
     headers,
   });
 };
+export const getAcceleratedLabsAndTests = async timeFrame => {
+  const from = `start_date=${timeFrame}`;
+  const to = `&end_date=${timeFrame}`;
+  return apiRequest(
+    `${API_BASE_PATH_V2}/medical_records/labs_and_tests?${from}${to}`,
+    {
+      headers,
+    },
+  );
+};
+
+export const getAcceleratedLabsAndTestsDetails = async id => {
+  return apiRequest(
+    `${API_BASE_PATH_V2}/medical_records/labs_and_tests/${id}`,
```

Comment:
since we don't have a way to get single results by ID, this call will fail. im good with just a (somewhat) unhandled error here for the time being assuming just the generic "not found" page is what will render?